sdk/typescript/rooch-sdk-kit/src/types/address/bitcoinAddress.ts

```diff
@@ -3,31 +3,26 @@
 
 import { base58_to_binary } from 'base58-js'
 import { bech32, bech32m } from 'bech32'
-import { sha3_256 } from '@noble/hashes/sha3'
+import { createHash } from 'sha256-uint8array'
+
+const sha256 = (payload: Uint8Array) => createHash().update(payload).digest()
 
 enum Network {
   mainnet = 'mainnet',
   testnet = 'testnet',
   regtest = 'regtest',
 }
 
+// The method used to distinguish bitcoin address payload type.
+// Ref: BitcoinAddressPayloadType https://github.com/rooch-network/rooch/blob/main/crates/rooch-types/src/address.rs
 enum AddressType {
-  p2pkh = 'p2pkh',
-  p2sh = 'p2sh',
-  p2wpkh = 'p2wpkh',
-  p2wsh = 'p2wsh',
-  p2tr = 'p2tr',
+  p2pkh = 0,
+  p2sh = 1,
+  p2wpkh = 2,
+  p2wsh = 2,
+  p2tr = 2,
```

Comment:
We shall call them all as [Bech32](https://en.bitcoin.it/wiki/Bech32) addresses.

Reply:
The method used to distinguish bitcoin address payload type.
It is currently used as a simple tag and is only used in rooch